packages/ui-components/src/components/Table/TableRow.js

```diff
@@ -46,11 +48,12 @@ export const TableRowCells = React.memo(({ columns, rowData, ExpandButton }) =>
         </TableCell>
       );
     },
-  ),
-);
+  );
+});
 
 TableRowCells.propTypes = {
   columns: PropTypes.arrayOf(PropTypes.shape(tableColumnShape)).isRequired,
+  rowProps: PropTypes.shape(PropTypes.shape(tableRowPropsShape)).isRequired,
```

Comment:
Didn't love having to do this, but there didn't seem to be another way to pass down properties to the actual row/cell level. The fact that the `rowProps` then need to be declared for all parent components was really quite a pain 😞 

Bit on the fence with this naming of `rowProps`. I wouldn't like `row`/`rows` as that would imply it's the actual data.

Reply:
Ah great call! Infact, I don't even need to add the `className` can just use `MuiTableCell-root`!